api/nodes/serializers.py

```diff
@@ -124,9 +128,126 @@ def update(self, instance, validated_data):
         return instance
 
 
+class NodeContributorsSerializer(JSONAPISerializer):
+    """ Separate from UserSerializer due to necessity to override almost every field as read only
+    """
+    filterable_fields = frozenset([
+        'fullname',
+        'given_name',
+        'middle_name',
+        'family_name',
+        'id',
+        'bibliographic',
+        'permissions'
+    ])
+    id = ser.CharField(source='_id', label='ID')
+    fullname = ser.CharField(read_only=True, help_text='Display name used in the general user interface')
```

Comment:
1. Consistency with all the other fields in the serializer
2. Readability of field labels

The UserSerializer now addresses point 2 by adding a "label" attribute as one alternative. You can bundle whatever change is made into [#OSF-4235]. I won't block this PR, but because bringing up to spec with UserSerializer will change the format of output data, I recommend that the serializer changes be made before the API is released.